master/internal/api_tensorboard.go

```diff
@@ -112,8 +105,8 @@ func (a *apiServer) GetTensorboard(
 		return nil, err
 	}
 
-	if ok, err := command.AuthZProvider.Get().CanGetNSC(
-		ctx, *curUser, model.UserID(resp.Tensorboard.UserId), command.PlaceHolderWorkspace); err != nil {
+	if ok, err := command.TbAuthZProvider.Get().CanGetTensorboard(
```

Comment:
Original intention was to have a single provider per module, i.e. one `command.AuthZProvider` for all "command" stuff.

Reply:
Yeah, and for the other NSCs we have a single interface but since the tensorboards case is a bit different, where it needs to consult with the experiment authz interface, this is introduced as one of the options. We (me and @eecsliu ) chatted about an alternative where I think we can delegate all of the tensorboard authorizations to `CanGetExperimentArtifacts` and maybe introduce a new method to do this in batch where we can use it for all the sources of a tensorbaord in one go. Eric would probably have more to add.

Reply:
but the methods on this interface can just be a part of methods of the shared "command" interface. it does not matter really if they have a different signature. using stuff from experiments could've been a deciding factor if it cause an import cycle, but then you'd have to split this go module anyway.

Reply:
ah ok I'll just merge this into the commands interface